pkg/process/statsd/statsd.go

```diff
@@ -28,7 +28,16 @@ func Configure(host string, port int, lookInEnv bool) error {
 		statsdAddr = net.JoinHostPort(host, strconv.Itoa(port))
 	}
 
-	client, err := statsd.New(statsdAddr)
+	options := []statsd.Option{
+		// Create a separate client for the telemetry to be sure we don't loose it.
+		statsd.WithTelemetryAddr(statsdAddr),
+		// Enable recommended settings to reduce the number of packets sent and reduce
+		// potential lock contention on the critical path.
+		statsd.WithChannelMode(),
+		statsd.WithClientSideAggregation(),
```

Comment:
Isn't the basic aggregation already enabled by default? https://github.com/DataDog/datadog-go/blob/master/README.md#basic-aggregation

Reply:
Yes it is, but I wanted to be a bit more explicit. Do you think this is an issue ?